providers/azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/DiskApiMockTest.java

```diff
@@ -78,6 +79,27 @@ public void createDiskWithTags() throws InterruptedException {
       assertTrue(dataDisk.tags().containsValue("jclouds-test-tag"));
    }
 
+   public void createDiskWithTagsAndSku() throws InterruptedException {
+
+      server.enqueue(jsonResponse("/creatediskwithtagsandskuresponse.json").setResponseCode(200));
+
+      final DiskApi diskApi = api.getDiskApi(resourcegroup);
+
+      DiskProperties properties = DiskProperties.builder().diskSizeGB(2).creationData(CreationData.create(CreationData.CreateOptions.EMPTY)).build();
+
+      DiskSku sku = DiskSku.builder().name("Premium_LRS").build();
```

Comment:
Is there an enum of the existing valid Disk types?

Reply:
According to azure docs only 4 values are allowed, I'll add an enum